core/src/test/scala/unit/kafka/controller/ControllerChannelManagerTest.scala

```diff
@@ -605,8 +610,11 @@ class ControllerChannelManagerTest {
     assertEquals(1, batch.sentRequests.size)
     assertTrue(batch.sentRequests.contains(2))
 
-    val requests = batch.collectStopReplicRequestsFor(2, expectedStopReplicaRequestVersion)
-    assertTrue(requests.forall(_.version() == expectedStopReplicaRequestVersion))
+    val requests = batch.collectStopReplicRequestsFor(2)
```

Comment:
Given that we've verified right above that the `sentRequests` are just one, should we change this assertion to asserting just one request?